src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/noLineWrap/InputNoLineWrapGood.java

```diff
@@ -0,0 +1,13 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.noLineWrap;
```

Comment:
i had my exams so could not work last week.........will just do these now.............